app/serializers/events_serializer.rb

```diff
@@ -0,0 +1,23 @@
+ class EventsSerializer
+
+   def initialize(events)
+     @events = events
+   end
+
+   def serialize_index
+     output = { "events": [] }
+     @sports ||= @events.select(:sport).distinct.pluck(:sport)
```

Comment:
Memoizing may not matter much here. The events table is really small overall, so although I'm not really excited about the calls I'm making to the database on this, it's still going to be relatively fast and the number of events in the olympics shouldn't vary widely (or increase a large amount) between games.